vars/Debian.yml

```
@@ -2,6 +2,7 @@
 
 __postgresql_apt_key_url: "https://www.postgresql.org/media/keys/ACCC4CF8.asc"
 __postgresql_apt_repo: "deb http://apt.postgresql.org/pub/repos/apt/ {{ ansible_distribution_release }}-pgdg main"
+__postgresql_apt_filename: "apt_postgresql_org_pub_repos_apt.list"

```

```suggestion
__postgresql_apt_filename: "apt_postgresql_org_pub_repos_apt"

```

The `.list` extension will be added automatically - https://github.com/ansible/ansible/blob/stable-2.11/lib/ansible/modules/apt_repository.py#L241